objects/src/accounts/mod.rs

```diff
@@ -58,13 +59,14 @@ impl Account {
     pub fn new(
         id: AccountId,
         storage_items: &[StorageItem],
+        merkle_store: MerkleStore,
```

Comment:
Since we are passing `merkle_store` as an owned type, should we do the same for `storage_items`? (i.e., pass them as a vector).

Another option could be to construct `AccountStorage` outside and pass a constructed storage object into the `Account` constructor.

Reply:
Yep makes sense. I think passing `AccountStorage` as a constructed object to the `Account` constructor makes the most sense. The constructor of `AccountStorage` should take an owned type for `storage_items`.